apps/ffdweb-site/next.config.mjs

```diff
@@ -4,6 +4,10 @@ import { redirects } from './redirects.mjs'
  * @type {import('next').NextConfig}
  */
 const imageRemotePatterns = [
+  {
+    protocol: 'https',
+    hostname: 'lh5-us.googleusercontent.com',
+  },
```

Comment:
Why do we use this? 😊

Reply:
[Remote image patterns](https://nextjs.org/docs/app/api-reference/components/image#remotepatterns) in Next.js are needed to specify which external domains are allowed as image sources for the `next/image` component, which helps with security by preventing arbitrary external images from being served through your application.